resources/compass/values.yaml

```diff
@@ -22,7 +22,7 @@ global:
       version: "ffe1a7c3"
     director:
       dir:
-      version: "95fc393e"
+      version: "56884fe1"
```

Comment:
What about schema migrator? I think it hasn't been bumped before

Reply:
Well, it has all required migrations for this component